### DISCUSSION TITLE
Trouble with Preprocessing the Data

### DISCUSSION THREAD

**Discussion by ssomani10** [WAM2layers/WAM2layers#216] [Q&A] [answered]
I am trying to work through the example case by preprocessing the data.  I had no trouble using the script to download the data, but when I tried to preprocess the data, I obtain  a few errors from the attached screenshot.

<img width="1683" alt="processing" src="https://user-images.githubusercontent.com/8172915/235816797-4cab9f95-4976-492d-934f-86615c4c309b.png">

**Comment by ruudvdent** [comment, marked as answer]
dear @ssomani10 , since I saw other questions already, I will assume this problem was solved? Please confirm.
